catroid/src/main/java/org/catrobat/catroid/ui/WebViewActivity.java

```diff
@@ -249,6 +255,29 @@ private String getExtensionFromContentDisposition(String contentDisposition) {
 		return extension;
 	}
 
+	public void setLoginCookies(String url) {
+		SharedPreferences sharedPreferences =
+				PreferenceManager.getDefaultSharedPreferences(getApplicationContext());
```

Comment:
If you declare sharedPreferences and cookieManager as a parameter, you'll be able to test the method. (Hint: mock it)